pallets/inflation-rewards/src/lib.rs

```diff
@@ -91,9 +92,26 @@ pub mod pallet {
 
             // Get the number of chains at this block (tanssi + container chain blocks)
             weight.saturating_accrue(T::DbWeight::get().reads_writes(1, 1));
-            let registered_para_ids = T::ContainerChains::current_container_chains();
+            let container_chains_to_check_unbounded: Vec<_> =
+                T::ContainerChains::container_chains_with_collators(ForSession::Current)
+                    .into_iter()
+                    .filter_map(|(para_id, collators)| (!collators.is_empty()).then_some(para_id))
+                    .collect();
+
+            // Convert to BoundedVec. If the number of chains is greater than the limit, truncate
+            // and emit a warning. This should never happen because we assume that
+            // MaxContainerChains has the same value in all the pallets, but that's not enforced.
+            // A better solution would be for container_chains_with_collators to return an already
```

Comment:
exactly!

Reply:
For the future because other pallets use this trait assuming it returns a `Vec`, so changing to `BoundedVec` will break all the tests that use `vec!` syntax.